src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/ResolveElasticsearchStep.java

```diff
@@ -110,4 +115,14 @@ protected File getUnpackDirectory()
         return upackDirectory;
     }
 
+    private void keepExistingDataCheck(InstanceConfiguration config) throws IOException {
```

Comment:
Please rename the method. Just by looking at the name, one would not know that it modifies data. I suggest something like removeExistingData.

On a separate note, please use the existing code style: braces on new line.